css/main.css

```diff
@@ -27,28 +27,100 @@ header {
   color: white;
 }
 
-#desktop-nav, #desktop-nav ul {
+li {
+  list-style: none;
+  font-family: 'Inter', sans-serif;
+  font-style: normal;
+  font-weight: normal;
+  font-size: 22px;
+  line-height: 43px;
+  text-align: center;
+  color: #fff;
+  padding: 5px;
+}
+
+.desktop-nav {
   display: none;
 }
 
-#logo {
-  font-family: "Roboto", sans-serif;
-  font-style: normal;
-  font-weight: bold;
-  font-size: 18px;
-  line-height: 24px;
+.desktop-nav.visible {
+  display: flex;
+  position: fixed;
+  top: 0;
+  left: 0;
+  background-color: #3c3a39;
+  min-height: 30vh;
+  z-index: 999;
+  width: 100%;
+}
```

Comment:
You did an amazing job but please fix the design of the mobile menu because is it not like in the Figma [design](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=1%3A1471) and it was part of the requirements. Keep coding! 💪